### PR TITLE
server: change unit of heartbeat ts to ms

### PR DESCRIPTION
**Pull request by Connor1996**

### CODE REVIEW

File: server/grpc_service.go [hunk 1]
```diff
@@ -342,7 +342,7 @@ func (s *Server) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error {
 		storeLabel := strconv.FormatUint(storeID, 10)
 
 		regionHeartbeatCounter.WithLabelValues(storeLabel, "report", "recv").Inc()
-		regionHeartbeatLatency.WithLabelValues(storeLabel).Observe(float64(time.Now().Second()) - float64(request.GetTimestamp()))
+		regionHeartbeatLatency.WithLabelValues(storeLabel).Observe(float64(time.Now().Nanosecond()/1000000) - float64(request.GetTimestamp()))
```
Comment (disksing) [inline server/grpc_service.go]: 1000000 -> time.Millisecond

File: server/grpc_service.go [hunk 2]
```diff
@@ -342,7 +342,7 @@ func (s *Server) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error {
 		storeLabel := strconv.FormatUint(storeID, 10)
 
 		regionHeartbeatCounter.WithLabelValues(storeLabel, "report", "recv").Inc()
-		regionHeartbeatLatency.WithLabelValues(storeLabel).Observe(float64(time.Now().Second()) - float64(request.GetTimestamp()))
+		regionHeartbeatLatency.WithLabelValues(storeLabel).Observe(float64(time.Now().Nanosecond()/int(time.Millisecond)) - float64(request.GetTimestamp()))
```
Comment (disksing) [inline server/grpc_service.go]: Nanosecond -> UnixNano

Comment (siddontang) [reply server/grpc_service.go]: why changing here?  Mostly, seconds is preferred.

Comment (disksing) [reply server/grpc_service.go]: I think using second is too inaccurate, in most cases the delay will be jumping between 0 and 1.

Comment (siddontang) [reply server/grpc_service.go]: why? you can set 0.001 as the base unit.

Comment (siddontang) [reply server/grpc_service.go]: Mostly, we use 0.0005

Comment (disksing) [reply server/grpc_service.go]: It is not the metrics we are updating but the accuracy of time that tikv sends to pd.

**Pull request merged**